docker/nginx/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive \
+     TZ=UTC \
+      apt-get install --no-install-recommends --no-install-suggests -y \
+        build-essential autoconf libtool pkg-config ca-certificates \
+        cmake gcc g++ python3 git nginx libpcre3-dev
+
+RUN git clone --shallow-submodules --depth 1 --recurse-submodules -b v1.35.0 https://github.com/grpc/grpc \
+  && cd grpc \
+  && mkdir -p cmake/build \
+  && cd cmake/build \
+  && cmake -DgRPC_INSTALL=ON -DgRPC_BUILD_TESTS=OFF -DgRPC_BUILD_CSHARP_EXTENSIONS=OFF ../.. \
+  && make -j2 \
+  && make install
+
+RUN git clone --shallow-submodules --depth 1 --recurse-submodules \
```

```suggestion
RUN git clone --shallow-submodules --depth 1 -b "v0.4.0" --recurse-submodules \
```
we need to pin the versions of the opentelemetry-cpp and opentelemetry-cpp-contrib repos as they diverged in the meantime which leads to a failing build